master/internal/db/postgres.go

```diff
@@ -1919,3 +1919,46 @@ WHERE t.id = $1;
 `, id)
 	return errors.Wrapf(err, "error updating best validation for trial %d", id)
 }
+
+// RunResourceAllocationAggregation updates the aggregated resource allocation table.
+func (db *PgDB) RunResourceAllocationAggregation() error {
```

Comment:
```suggestion
func (db *PgDB) UpdateResourceAllocationAggregations() error {
```
Maybe to make it more clear this is expensive and may touch multiple rows?